frontend/packages/topology/integration-tests/features/topology/topology-editing-app-node.feature

```diff
@@ -50,3 +50,155 @@ Feature: Editing an application
+
+
+        @regression @manual
+        Scenario: Edit form for deployment
+            Given user has created deployment workload
+              And user is at the Topology page
+             When user right clicks on the node to open context menu
+              And user selects option "Edit Deployment" from context menu
+             Then user will see option to switch between YAML view and Form view
+              And user will see Recreate Strategy type under Deployment Strategy selected by default
+              And user will see "Maximum number of unavailable Pods" and "Maximum number of surge Pods" sections for Strategy type RollingUpdate under Deployment Strategy
+              And user will see "Deploy image from an image stream tag" checkbox under Images with project, image stream and tag section associated with it
+              And user will see Image name section under Images with checkbox "Deploy image from an image stream tag" unchecked
+              And user will see "Show advanced image options" and "Show advanced container options" in case of image stream option
+              And user will see advanced option for "Pause rollouts" and "Scaling"
+
+
+        @regression @manual
+        Scenario: Edit form for deployment config
+            Given user has created deployment config workload
+              And user is at the Topology page
+             When user right clicks on the node to open context menu
+              And user selects option "Edit DeploymentConfig" from context menu
+             Then user will see option to switch between YAML view and Form view
+              And user will see "Timeout" section assciated with Recreate Strategy type under Deployment Strategy selected by default
+              And user will see "Timeout", "Maximum number of unavailable Pods" and "Maximum number of surge Pods" sections for Strategy type RollingUpdate under Deployment Strategy
+              And user will see "Image name", "Command" and "Environment variables(runtime only)" sections for Strategy type Custom under Deployment Strategy
+              And user will see "Deploy image from an image stream tag" checkbox under Images with project, image stream and tag section associated with it
+              And user will see Image name section under Images with checkbox "Deploy image from an image stream tag" unchecked
+              And user will see "Show additional parameters and lifecycle hooks", "Show advanced image options" and "Show advanced container options" in case of image stream option
+              And user will see advanced option for "Pause rollouts" and "Scaling"
+
+
+        @regression @manual
+        Scenario: Additional parameters and lifecycle hooks for deployment config
+            Given user is on Edit DeploymentConfig page
+             When user clicks on "Show additional parameters and lifecycle hooks" option
```

Comment:
how can user clicks on 2 links at same point of time?

Reply:
That's a single link. Its complete name has `and` in it